### PR TITLE
Rename atomic wait/notify instructions

### PR DESCRIPTION
**Pull request by aheejin**
This renames the following:
- `i32.wait` -> `i32.atomic.wait`
- `i64.wait` -> `i64.atomic.wait`
- `wake` -> `atomic.notify`

to match the [spec](https://github.com/WebAssembly/threads/blob/master/proposals/threads/Overview.md#wait-and-notify-operators).

### CODE REVIEW

File: test/atomics.wast [hunk 1]
```diff
@@ -134,24 +134,24 @@
    )
   )
  )
- (func $atomic-wait-wake (type $0)
+ (func $atomic-wait-atomic.notify (type $0)
```
Comment (tlively) [inline test/atomics.wast]: This name seems a little strange now. Maybe `atomic-wait-notify`?

Comment (aheejin) [reply test/atomics.wast]: Done. Auto search-replace was the culprit :$

File: src/wasm-traversal.h [hunk 2]
```diff
@@ -473,7 +473,7 @@ struct Walker : public VisitorType {
   static void doVisitAtomicRMW(SubType* self, Expression** currp)    { self->visitAtomicRMW((*currp)->cast<AtomicRMW>()); }
   static void doVisitAtomicCmpxchg(SubType* self, Expression** currp){ self->visitAtomicCmpxchg((*currp)->cast<AtomicCmpxchg>()); }
   static void doVisitAtomicWait(SubType* self, Expression** currp)   { self->visitAtomicWait((*currp)->cast<AtomicWait>()); }
-  static void doVisitAtomicWake(SubType* self, Expression** currp)   { self->visitAtomicWake((*currp)->cast<AtomicWake>()); }
+  static void doVisitAtomicNotify(SubType* self, Expression** currp)   { self->visitAtomicNotify((*currp)->cast<AtomicNotify>()); }
```
Comment (kripken) [inline src/wasm-traversal.h]: indentation of the `{` compared to other lines should stay consistent as beore

Comment (aheejin) [reply src/wasm-traversal.h]: Done

**Pull request merged**